src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
+
+	@Override
+	public boolean checkValidity(CreditCardVendor ccv, String number) {
```

Comment:
1. Use "final" modifiers by default  - it will express that the method doesn't plan to change the declared variable.
2. Add some generic (vendor-independent) checks ( blank/null number,  spaces at begining/end etc.)